index.html

```diff
@@ -7,10 +7,70 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
+
 </head>
 
 <body>
     <!-- Add your markup here -->
+    <div class="flex-container">
+        <div>LOGO</div>
+        <div>CONTACT</div>
+     
+
+    </div>
+
+<br>
```

Comment:
While it's valid HTML, generally it's better to use css padding and margin instead of <br> to create spacing.